examples/render/contour.yaml

```diff
@@ -209,7 +72,965 @@ data:
+                            to since a service can have multiple defined
+                          type: integer
+                        strategy:
+                          description: LB Algorithm to apply (see https://github.com/projectcontour/contour/blob/master/design/ingressroute-design.md#load-balancing)
+                          type: string
+                        validation:
+                          description: UpstreamValidation defines how to verify the
+                            backend service's certificate
+                          properties:
+                            caSecret:
+                              description: Name of the Kubernetes secret be used to
+                                validate the certificate presented by the backend
+                              type: string
+                            subjectName:
+                              description: Key which is expected to be present in
+                                the 'subjectAltName' of the presented certificate
+                              type: string
+                          required:
+                          - caSecret
+                          - subjectName
+                          type: object
+                        weight:
+                          description: Weight defines percentage of traffic to balance
+                            traffic
+                          format: int32
+                          type: integer
+                      required:
+                      - name
+                      - port
+                      type: object
+                    type: array
+                  timeoutPolicy:
+                    description: The timeout policy for this route
+                    properties:
+                      request:
+                        description: Timeout for receiving a response from the server
+                          after processing a request from client. If not supplied
+                          the timeout duration is undefined.
+                        type: string
+                    required:
+                    - request
+                    type: object
+                required:
+                - match
+                type: object
+              type: array
+            tcpproxy:
+              description: TCPProxy holds TCP proxy information.
+              properties:
+                delegate:
+                  description: Delegate specifies that this tcpproxy should be delegated
+                    to another IngressRoute
+                  properties:
+                    name:
+                      description: Name of the IngressRoute
+                      type: string
+                    namespace:
+                      description: Namespace of the IngressRoute. Defaults to the
+                        current namespace if not supplied.
+                      type: string
+                  required:
```

Comment:
both and name and namespace are required for IngressRoute.tcpproxy.delegate.

Reply:
I'll fix the comment too then.

Reply:
turns out namespace is not required. Leaving this one how it is.